### ISSUE TITLE
Custom CRDs are not applied by default

### ISSUE THREAD

**Issue by dz-pyps** [pulumi/pulumi-kubernetes#1144]
### Problem description

This is a follow-up for the issue https://github.com/pulumi/pulumi-kubernetes/issues/1023.

Using **Python-SDK** and **pulumi-kubernetes**, I was unable to get custom CRDs applied by default.

I used Traefik Helm chart https://github.com/containous/traefik-helm-chart

### Errors & Logs

```
Diagnostics:
  kubernetes:traefik.containo.us:IngressRoute (traefik-dashboard):
    error: creation of resource default/traefik-dashboard failed because the Kubernetes API server reported that the apiVersion for this resource does not exist. Verify that any required CRDs have been created: no matches for kind "IngressRoute" in version "traefik.containo.us/v1alpha1"
```

### Affected product version(s)

Pulumi version: **v2.2.1**
Helm version:  **{Version:"v3.2.1", GitCommit:"fe51cd1e31e6a202cba7dead9552a6d418ded79a", GitTreeState:"clean", GoVersion:"go1.13.10"}**
SDK: **Python**
Pulumi-kubernetes version: **pulumi-kubernetes==2.2.0**

**Comment by jaxxstorm** [comment]
Fix for this incoming

**Comment by lblackstone** [comment]
Looks like this was fixed in #1145
